data_integration/logging/run_log.py

```diff
@@ -65,6 +66,8 @@ class SystemStatistics(Base):
     swap_usage = sqlalchemy.Column(sqlalchemy.FLOAT)
     iowait = sqlalchemy.Column(sqlalchemy.FLOAT)
 
+    __table_args__ = (sqlalchemy.UniqueConstraint('timestamp', 'run_id', name='timestamp+run_id'), )
```

Comment:
I think this is not needed: with the current version as implemented here, the primary key will fail even as the unique constrain is fine in that case. When this is switched to have both columns as primary key, this would need to be removed again. So I think up to that point we do not need it and afterwards we also do not as the primary key is then taking over.

Reply:
It's reasonable!